session/bootstrap.go

```diff
@@ -240,6 +240,7 @@ const (
 		update_time timestamp(3) NOT NULL,
 		charset text NOT NULL,
 		collation text NOT NULL,
+		create_way varchar(20) NOT NULL default 'unknow',
```

Comment:
```suggestion
		create_way varchar(10) NOT NULL default 'unknown',
```

After we execute the `alter column` in bootstrap, it's better not to create the column when create table.

Reply:
@lzmhhh123  `upgradeToVer47` will only be done when the version code is lower than 47, while current version is 47, so create the column when create table is necessary.
I checked the implementation of other `upgradeToVerX` functions, they did it in the same way.

Reply:
for instance, `upgradeToVer29` adds an index, and relevant index is created when create table, too.

Reply:
@lzmhhh123  and , `sql_created` is more than 10 chars, so we need varchar(20). 😆 (I didn't realized it first, either.)